flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/data/FlinkParquetReaders.java

```diff
@@ -321,6 +327,29 @@ public DecimalData read(DecimalData ignored) {
     }
   }
 
+  private static class TimestampInt96Reader extends ParquetValueReaders.UnboxedReader<Long> {
+    private static final long UNIX_EPOCH_JULIAN = 2_440_588L;
+
+    TimestampInt96Reader(ColumnDescriptor desc) {
+      super(desc);
+    }
+
+    @Override
+    public Long read(Long ignored) {
+      return readLong();
+    }
+
+    @Override
+    public long readLong() {
```

Comment:
This will require some tests.

Reply:
Ok， Thanks for your advice

Reply:
As for the test , I have one idea:
because in file  `flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/data/TestFlinkParquetReader.java`   has one test case `testInt96TimestampProducedBySparkIsReadCorrectly`,   I  add another method like below: 
```java
  protected List<RowData> rowDatasFromFile(InputFile inputFile, Schema schema) throws IOException {
    try (CloseableIterable<RowData> reader =
                 Parquet.read(inputFile)
                         .project(schema)
                         .createReaderFunc(type -> FlinkParquetReaders.buildReader(schema, type))
                         .build()) {
      return Lists.newArrayList(reader);
    }
  }
```

and add this code
```java
    List<RowData> readDataRows = rowDatasFromFile(parquetInputFile, schema);
    Assert.assertEquals(rows.size, readRows.size());
```
after the spark `rowsFromFile` check .

Reply:
flinkv1.14  readLong test has added,  could u please review it  ?  @rdblue  @kbendick

Reply:
Finally  I added the support to both flink v1.13 and v1.14,  and  for the test
- add test case in  flink v1.13 , flink v1.14 module
- for generate parquet-int96  as `TestSparkParquetReaders` does,  import `spark-sql` in  `testImplementation` of build.gradle
- just use  `GenericInternalRow` and `RandomUtil.generatePrimitive` to generate `List<InternalRow> rows` that will be written to parquet file as int96 timestamp type

how about this ? @rdblue @kbendick